rps.js

```diff
@@ -0,0 +1,22 @@
+const playGame = function (rock){
```

Comment:
this program only tests for rock, and the output is not in the desired form it should be "Computer picked: paper. User picked: rock. User lose." while this code prints either "win" or "lose", nothing about draw